### PR TITLE
MSBuild: automerges for vs17*->main

### PR DESCRIPTION
**Pull request by rainersigwald**
This is what our release branches will likely be so setting up the automatic RI.

### CODE REVIEW

**Comment by rainersigwald** [comment]
@mmitche can you approve this when you get a chance? Or give me the commit bit 😇

**Review by mmitche — APPROVED** [review]

**Pull request merged**
